src/SearchButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const SearchButton = () => {
+  return (
+    <div>
```

Comment:
Why did you add an extra div around the button? It's not wrong, but I'm also not sure it's necessary, and generally when coding less is better :)